installation/install

```diff
@@ -115,21 +112,24 @@ function sys_upgrade
 		return
 	fi
 	case "$os" in
-		opensuse)
-			sudo zypper update
-			;;
 		archlinux)
 			sudo pacman -Syu
 			;;
-		fedora)
-			sudo dnf update
-			;;
 		debian)
 			sudo apt-get update; sudo apt-get upgrade
 			;;
+		fedora)
+			sudo dnf update
+			;;
 		gentoo)
 			sudo emerge -u world
 			;;
+		opensuse)
+			sudo zypper update
+			;;
```

Comment:
Same comment as above on the movement of lines, not related to you issue

If you want to move or refactor codes, you should open an other issue for that, and we will discuss it